### PR TITLE
Create disk_resize.py

### PR DESCRIPTION
**Pull request by sahilmgandhi**
Disk resizing sample code to be used as a sample :)

### CODE REVIEW

**Comment by sahilmgandhi** [comment]
Hi, could an admin look at this (and other) pull requests please.

File: samples/disk_resize.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    task = vm_obj.ReconfigVM_Task(spec=spec)
+    tasks.wait_for_tasks(si, [task])
+    return True
+
+
+def get_args():
+
+    # Parse through the arguments. Style and format taken from the other
+    # samples in the samples directory
+    parser = argparse.ArgumentParser()
```
Comment (tianhao64) [inline samples/disk_resize.py]: Most of these arguments are already handled in the cli.py helper class. Can you use it instead.

File: samples/disk_resize.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    vm_disk_spec.device.capacityInKB = size * 1024 * 1024
+
+    # Actually go and change the vm disk
+    dev_changes = []
+    dev_changes.append(vm_disk_spec)
+    spec = vim.vm.ConfigSpec()
+    spec.deviceChange = dev_changes
+    task = vm_obj.ReconfigVM_Task(spec=spec)
+    tasks.wait_for_tasks(si, [task])
+    return True
```
Comment (tianhao64) [inline samples/disk_resize.py]: why return true?

Comment (khalilnawar) [reply samples/disk_resize.py]: this script for  any version of VCenter?

**Comment by sahilmgandhi** [comment]
@tianhao64 can you take a look again?

**Comment by sahilmgandhi** [comment]
I unfortunately do not have context on this project anymore, nor would I be able to reproduce it easily, so am closing out this pull request.

**Pull request closed without merging**